torchci/lib/bot/ciflowPushTrigger.ts

```diff
@@ -133,54 +136,24 @@ async function handleLabelEvent(context: Context<"pull_request.labeled">) {
   if (!isCIFlowLabel(label)) {
     return;
   }
-
-  // collected from .github/workflows/*
-  const valid_labels = [
-    "ciflow/trunk",
-    "ciflow/periodic",
-    "ciflow/android",
-    "ciflow/binaries",
-    "ciflow/unstable",
-    "ciflow/inductor",
-    "ciflow/mps",
-    "ciflow/nightly",
-    "ciflow/binaries_conda",
-    "ciflow/binaries_libtorch",
-    "ciflow/binaries_wheel",
-    "ciflow/inductor-perf-test-nightly",
-  ];
+  const config: any = await tracker.loadConfig(context);
+  const valid_labels: Array<string> =
+    config !== null ? config["ciflow_push_tags"] : null;
+  if (valid_labels == null) {
+    await context.octokit.issues.createComment(
+      context.repo({
+        body: "No ciflow labels are configured for this repo.",
```

Comment:
A link to some documentation on how one can create a ciflow config would be a good quality of life improvement here

Reply:
https://github.com/pytorch/test-infra/wiki/PyTorch-bot#ciflow-bot